src/apicast-cli/rockspec.lua

```diff
@@ -0,0 +1,96 @@
+local path = require "pl.path"
+local persist = require "luarocks.persist"
+local type_check = require "luarocks.type_check"
+
+local _M = {}
+
+local function configure_lua_version(rockspec, luaver)
```

Comment:
This is not necessary. The lua version dependency should not be in the rock spec.